### ISSUE TITLE
ForceReply feature's logic needs update

### ISSUE THREAD

**Issue by W3LLD0NE** [tdlib/td#1277]
When user has multiple devices, and the bot sends him message with ForceReply - reply interface is replicated on each device. This is understandable and perfectly ok, because user can respond on any device.
BUT when user responds to the message on one device, reply interface is cleared ONLY on this device. It is STILL ATTACHED to other devices.
User then should manually clear reply interface on each his device, which is really annoying.
Please fix that. Add option either to auto-remove ForceReply from the message (on all devices) after the first user's answer, or to edit the message and remove it manually (you already have this option to remove inline keyboard, but it doesn't work with ForceReply).

**Comment by levlam** [comment]
This is expected behavior. If the reply interface needs to be removed, then the bot should send a message with `ReplyKeyboardRemove` after it received a reply from the user.

**Comment by W3LLD0NE** [comment]
> This is expected behavior. If the reply interface needs to be removed, then the bot should send a message with `ReplyKeyboardRemove` after it received a reply from the user.

My bad then. I've tried it previously as well, without success. Now I can say it is just a bug with iOS Telegram client specifically - it doesn't hide ForceReply after ReplyKeyboardRemove message received. That's why I thought it's broken generally because I didn't test on other platforms. Now I did and it works on Windows & Android, but not on iOS. Closed.
